src/main/java/mate/service/impl/DriverServiceImpl.java

```diff
@@ -0,0 +1,57 @@
+package mate.service.impl;
+
+import java.util.List;
+import mate.dao.DriverDao;
+import mate.lib.Inject;
+import mate.lib.Service;
+import mate.model.Driver;
+import mate.service.DriverService;
+
+@Service
+public class DriverServiceImpl implements DriverService {
+    @Inject
+    private DriverDao driverDao;
+
+    @Override
+    public Driver create(Driver driver) {
+        List<Driver> drivers;
+        drivers = driverDao.getAll();
+        for (Driver driverItem : drivers) {
+            if (driver.getLicenseNumber().equals(driverItem.getLicenseNumber())) {
+                return driverItem;
+            }
+        }
+        return driverDao.create(driver);
+    }
+
+    @Override
+    public Driver get(Long id) {
+        if (id == null) {
+            return new Driver();
+        }
+        return driverDao.get(id).orElse(new Driver());
```

Comment:
```suggestion
        return driverDao.get(id).get();
```

Если id == null у нас вернется пустой Optional и вылетит ошибка при попытке достать водителя тут `.get()`. Плохая практика возвращать `new Driver()`, если такого водителя в базе нету. Лучше что б ошибка вылетела в месте ее возникновения, чем в каком-то другом месте кода